ccd-definition/CaseEventToFields/CareSupervision/uploadC2-prod.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "uploadC2",
+    "CaseFieldID": "temporaryC2Document",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "COMPLEX",
+    "PageID": 2,
+    "PageDisplayOrder": 2,
+    "PageColumnNumber": 1,
+    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/upload-c2/validate-pba-number/mid-event"
```

Comment:
So by looking at the controller it seems you renamed that URL, this means the code you are pushing for prod is going to fail as there is no longer this endpoint. You either need to have two endpoints, or implement feature toggle to switch the code, or implement the code which would work for two flows. 

Or am I missing something? But I don't see this endpoint anymore... Remember merging the code pushes that code to prod.

Reply:
you are right, I should not delete this backend code. I have put that back.